addons/website/static/src/scss/website.ui.scss

```diff
@@ -20,16 +20,23 @@ $-mini-nav-size: 40px;
         cursor: pointer;
     }
 
-    &::before, .o_frontend_to_backend_icon, .o_frontend_to_backend_apps_btn, .o_frontend_to_backend_edit_btn {
+    &::before, .o_frontend_to_backend_icon, .o_frontend_to_backend_buttons {
         transform-origin: top left;
-        transition: transform 400ms ease 0s;
+        transition: transform 800ms ease 500ms;
```

Comment:
Strange you double the duration but it appears as if it is the same :shrug: Maybe an illusion... but I would speed up the closing a bit and increase the delay a bit more as well.